src/pages/dashboard/components/Activity.tsx

```diff
@@ -25,7 +25,7 @@ export function Activity() {
   const { data, isLoading, isError } = useQuery(
     ['/api/v1/activities'],
     () => request('GET', endpoint('/api/v1/activities?reactv2')),
-    { staleTime: 60 }
+    { staleTime: 300000 }
```

Comment:
@beganovich @turbo124 and I had a quick discussion in the private chat, so now we will invalidate the activities query when payments, invoices, or quotes are invalidated OR every 5 minutes.